splink/profile_data.py

```diff
@@ -117,6 +158,60 @@ def _get_df_percentiles():
     return sqls
 
 
+def _get_df_kde():
+    sql = """
+    select
+        value,
+        value_count,
+        group_name,
+    from __splink__df_all_column_value_frequencies
+    """
+    return sql
+
+
+def _get_df_correlations(column_expressions):
```

Comment:
I would probably split up your CTE expressions first and then attempt to run the code above in debug mode. That should help you narrow down what's going wrong in the background.

This will ultimately all come down to a missing feature of postgres, but I'm not familiar enough with the db to be able to give you any clues.

Reply:
Thanks for diagnosing. Code was actually left in in error as this PR was originally meant to be larger, but was cut down. Have removed this code but will save this comment so that I can refer back when doing the other PR.